examples/dem/collapsing_sand_pile_3d.jl

```diff
@@ -0,0 +1,94 @@
+using TrixiParticles
+using OrdinaryDiffEq
+
+# Physical parameters
+gravity = -9.81
+acceleration = (0.0, 0.0, gravity)
```

Comment:
Why is height the z coordinate in this example, while it's the y coordinate in all other examples?